packages/react-components/src/lib/cesium-map/tools/draw/drawHelper.ts

```diff
@@ -1012,6 +1015,20 @@ var DrawHelper = (function () {
     ]);
   }
 
+  // ALEX add to change draw mode cursor
```

Comment:
can you remove the comment?

Reply:
Sorry, here prefer to leave , it's mostly outer source, want keep ref. where I change the things

Reply:
than add TODO: instead of alex

Reply:
It's definitely not TODO, it's performed changes MC_CHANGES instead ALEX?